### DISCUSSION TITLE
Cannot modify header of template excel file from wellplate

### DISCUSSION THREAD

**Discussion by Konrad1991** [ComPlat/chemotion_ELN#1463] [Ideas]
- I am an ELNer

# Expected Behavior

After downloading the template excel file from a wellplate it is possible to change the header names. Next the modified version (lower image) can be uploaded into the wellplate and the results can be imported to the *List* tab. 

![image](https://github.com/ComPlat/chemotion_ELN/assets/35558461/d7ec11b7-4a5a-4ba7-8f47-2430735cf6eb)

![image](https://github.com/ComPlat/chemotion_ELN/assets/35558461/8ccc8d87-7e65-424b-84b6-a08d62956416)

# Current Behavior

When trying to import into the *List* tab the following error is shown:

![image](https://github.com/ComPlat/chemotion_ELN/assets/35558461/9e4b9da5-356d-4a69-945a-0d5d4a6dbab1)

## Failure Logs / Screenshots

Include any relevant log snippets or files here.

**Comment by dpatelev** [comment]
@Konrad1991 It looks like that the header name must have an underscore '_Value', in it, otherwise the import fails. This is a requirement of the excel file. Trying "Absorption_Value" should fix the problem - if it works, please report back :)

I think the header name requirement is mentioned in the wellplate file, but as I am not the one who created it, I am not sure.

**Comment by Konrad1991** [comment]
@endevor-music I'm aware that one needs Name_Value followed by Name_Unit. However, currently, the requirement came up to give each column independent names. Furthermore, the constraint of adding _Value and _Unit is not desired.
